### ISSUE TITLE
Some banner patterns are impossible / very difficult to obtain

### ISSUE THREAD

**Issue by Ameliorate** [CivClassic/AnsibleSetup#137]
[Banner Patterns](https://minecraft.gamepedia.com/Banner_Pattern) are used in a [Loom](https://minecraft.gamepedia.com/Loom) to add some patterns to banners in recent versions of minecraft.

The following patterns are unobtainable in civclassics:
* Globe (from villager trading)
* Snout (from piglin trading)
* Creeper face (maybe) (requires creeper head)

The following patterns are possibly a little too hard to obtain:
* Skeleton skull and crossbones (made with a wither skull)
* Mojang logo (made with enchanted golden apple, from mineshafts only at SOTW, very rare nowadays)
* Daisy (made with oxeye daisy, hard to get on some parts of the map)

In other words, all the banner patterns are perhaps too hard to obtain. 

In my opinion, the hard recipes should be made easier because banners are purely a buildfag item. There is no gameplay advantage to having a rare banner, besides bragging rights. However, most people don't understand the difficulty in obtaining a rare pattern. Making it easier to obtain these banner patterns would lead to more creative designs for banners and flags.

Perhaps a banner factory should be added, or recipes for banner patterns should be added to an existing factory.

**Comment by wingzero54** [comment]
Banner patterns are meant to be a little hard to obtain, they're a random laundry list of obscure items. That being said, maybe we can come up with recipes for Snouts and Globes since trading is disabled. The other option is making all the banner patterns as fossil drops, since we can do that now and not need the ingredient item.

**Comment by Ameliorate** [comment]
#148 fixes my complaints for the most part. I disagree with the chosen recipes, but that's a hard argument to make when I'm not the one writing the config.
